tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -85,8 +85,8 @@ public static IEnumerable<object[]> GetData()
         {
             foreach (var version in PackageVersions.OpenTelemetry)
             {
-                yield return new object[] { version[0], false };
-                yield return new object[] { version[0], true };
+                yield return new object[] { version[0] };
+                yield return new object[] { version[0] };
             }
         }
```

Comment:
nit: you can replace this whole method now with

```csharp
        public static IEnumerable<object[]> GetData() => PackageVersions.OpenTelemetry;
```

Reply:
Thanks done!